apps/frontend/src/app/PageTeam/CharacterDisplay/Tabs/TabOptimize/index.tsx

```diff
@@ -714,9 +797,16 @@ export default function TabBuild() {
             }}
             disabled={generatingBuilds}
           >
-            {/* TODO: Translation */}
-            Use artifacts in teammates' builds
+            <Box display="flex" gap={1}>
+              {/* TODO: Translation */}
+              <span>Use artifacts in teammates' active builds</span>
+              <strong>
+                {useTeammateBuild ? teammateArtifactIds.length : 0}/
```

Comment:
Technically this length isn't right. You need to use the tally logic we have in the other counters for if these artifacts are further included by the other filters. Could either fix it or remove the tally for now?

Reply:
Also the tally isn't styled the same as our other oens (i.e. inside a SqBadge)